custom/icds_reports/reports/prevalence_of_stunting.py

```diff
@@ -33,7 +33,7 @@ def get_data_for(filters):
         if not show_test:
             queryset = apply_exclude(domain, queryset)
         if 'age_tranche' not in config:
-            queryset = queryset.exclude(age_tranche=72)
+            queryset = queryset.exclude(age_tranche__lt=72)
```

Comment:
This will exclude all children before 72 months of age. you should also change `exclude` to `filter` which will mean it will say `include children of age less than 72 months `

Reply:
Done with the changes and updated the repository. Please have a look.